runtime/doc/news.txt

```diff
@@ -189,6 +189,10 @@ The following new APIs or features were added.
   disabled by default and can be enabled by setting the
   `workspace.didChangeWatchedFiles.dynamicRegistration=true` capability.
 
+• Treesitter injection queries now use the format described at
+  https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-injection .
+  Support for the previous format will be removed in a future release.
```

Comment:
This is a strong signal that this is the right change. When you can just point to upstream spec and say "we support [this]" instead of "[big paragraph about differences]", that is a gain for both developers and users.

An early, analogous case was in 2014 when we debated a variation of msgpack-rpc. https://github.com/neovim/neovim/pull/1121